tests/test_cli.py

```diff
@@ -172,3 +176,143 @@ def test_version(self):
         neofs_adm.version.get()
 
         shell.exec.assert_called_once_with(f"{self.neofs_adm_exec_path}   --version")
+
+    def test_shards_flush_cache(self):
```

Comment:
I would suggest to move these tests into a separate class/file/module/package. This TestCli looks messy.

Reply:
https://github.com/nspcc-dev/neofs-testlib/issues/77